README.md

```diff
@@ -213,6 +217,16 @@ CDC including this GitHub page may be subject to applicable federal law, includi
 
 #### ReportStream Setup
 
+### Generating and using a token
```

Comment:
This is a higher level heading than the one previous (`ReportStream Setup`) which turns it into a orphan.  Did you mean to put this new heading under the previous heading?